### PR TITLE
AMB-09 Create Project and ProjectProperties class models

### PR DESCRIPTION
**Pull request by Aaron-Millen**

### CODE REVIEW

**Review by CharlotteMKainos — COMMENTED** [review]
change float to double

**Review by CharlotteMKainos — APPROVED** [review]

File: src/main/java/org/example/models/Project.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+public class Project {
+    private int projectId;
+    private String name;
+    private int technologyId;
+    private int techLeadId;
+    private int clientId;
+    private int salesEmployeeId;
+    private ProjectProperties projProps;
+    public Project(
```
Comment (lukemKainos) [inline src/main/java/org/example/models/Project.java]: space pls

File: src/main/java/org/example/models/Project.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+            final int salesEmployeeId,
+            final ProjectProperties projProps
+    ) {
+        this.projectId = projectId;
+        this.name = name;
+        this.technologyId = technologyId;
+        this.techLeadId = techLeadId;
+        this.clientId = clientId;
+        this.salesEmployeeId = salesEmployeeId;
+        this.projProps = projProps;
```
Comment (lukemKainos) [inline src/main/java/org/example/models/Project.java]: proj = project

File: src/main/java/org/example/models/Project.java [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+    public int getTechnologyId() {
+        return technologyId;
+    }
+
+    public void setTechnologyId(final int technologyId) {
+        this.technologyId = technologyId;
+    }
+
+    public ProjectProperties getProjProps() {
```
Comment (lukemKainos) [inline src/main/java/org/example/models/Project.java]: proj?

File: src/main/java/org/example/models/ProjectProperties.java [hunk 4]
```diff
@@ -0,0 +1,55 @@
+package org.example.models;
+
+import org.joda.time.DateTime;
+
```
Comment (lukemKainos) [inline src/main/java/org/example/models/ProjectProperties.java:3]: double space

**Review by lukemKainos — CHANGES_REQUESTED** [review]

**Review by lukemKainos — APPROVED** [review]

**Pull request merged**